packages/perseus/src/__tests__/renderer.test.tsx

```diff
@@ -260,13 +263,13 @@ describe("renderer", () => {
             });
 
             // Assert
-            expect(sawDropdown2).toBeTrue();
+            expect(sawDropdown2).toBe(true);
```

Comment:
I started getting type errors on `.toBeTrue()` and `.toBeFalse()`. They appear to be from Jasmine and I'm unclear how this ever worked. 😦